### PR TITLE
Update logger help message content - short logs format

### PR DESCRIPTION
**Pull request by ktrzcinx**

### CODE REVIEW

File: developer_guides/debugability/logger/index.rst [hunk 1]
```diff
@@ -25,18 +25,18 @@ Logger usage
...
--t             Get traces from "/sys/kernel/debug/sof/trace" instead of from the default "/sys/kernel/debug/sof/etrace"
--p             Get traces from stdin instead of the default "/sys/kernel/debug/sof/etrace"
--c clock       Set the timestamp clock in MHz
--e             Enable checking the firmware version with the default verification file "/sys/kernel/debug/sof/fw_version"
--v ver_file    Enable checking the firmware version with the ver_file file instead of the default: "/sys/kernel/debug/sof/fw_version"
--s			       Take a snapshot of state
--r			       Less formatted output for chained log processors
--d			       Dump ldc information
+-h 		help
+-l ldc_file	Specify the ldc file
```
Comment (lgirdwood) [inline developer_guides/debugability/logger/index.rst:29]: "Specify the ldc file (debugging information and strings)"

File: developer_guides/debugability/logger/index.rst [hunk 2]
```diff
@@ -36,6 +36,7 @@ Usage sof-logger <option(s)> <file(s)>
 -v ver_file	Enable checking the firmware version with the ver_file file instead of the default: "/sys/kernel/debug/sof/fw_version"
 -s		Take a snapshot of state
 -r		Less formatted output for chained log processors
```
Comment (dbaluta) [inline developer_guides/debugability/logger/index.rst:38]: The commit description doesn't make sense for me. I think it is better to use the commit description to explain in detail what is the purpose of '-L'

File: developer_guides/debugability/logger/index.rst (hunk 1 above)
Comment (ktrzcinx) [reply developer_guides/debugability/logger/index.rst]: added

File: developer_guides/debugability/logger/index.rst (hunk 2 above)
Comment (ktrzcinx) [reply developer_guides/debugability/logger/index.rst:38]: updated

**Pull request merged**